### PR TITLE
Feature: Tokenized filter option

### LINKED ISSUE #53: Add Tokenized to Verify/Trees filter and Tree Detail
If a capture has a non-null `token_id` in the database, it is considered _Tokenized_.

We need to display whether or not a capture is Tokenized (in Tree Detail) and allow the user to filter on this value in the Verify and Trees pages (see #51 which aims to consolidate the filters on the two pages).

### PR DESCRIPTION
**Pull request by VLuisa**
Resolves #53 

Add 'Tokenization Status' filter to Verify and Capture pages with options to filter by 'All', 'Tokenized', and 'Not Tokenized'.

### CODE REVIEW

File: src/common/variables.js [hunk 1]
```diff
@@ -9,3 +9,7 @@ export const verificationStates = {
   AWAITING: 'Awaiting Verification',
   REJECTED: 'Rejected',
 };
+export const tokenizationStates = {
```
[the remark this answers is not in the document]
Comment (VLuisa) [reply src/common/variables.js:12]: Good point! Just addressed it

File: src/components/Filter.js [hunk 2]
```diff
@@ -251,6 +254,36 @@ function Filter(props) {
           </MenuItem>
         ))}
       </TextField>
+      <GSInputLabel text="Tokenization Status" />
+      <TextField
+        select
+        value={
```
Comment (nmcharlton) [inline src/components/Filter.js:260]: This can be simplified to `value={tokenId}`.

File: src/components/Filter.js [hunk 3]
```diff
@@ -251,6 +254,36 @@ function Filter(props) {
...
+          tokenId === filterOptionAll
+            ? filterOptionAll
+            : tokenId === tokenizationStates.TOKENIZED
+            ? tokenizationStates.TOKENIZED
+            : tokenizationStates.NOT_TOKENIZED
+        }
+        onChange={(e) => {
+          setTokenId(
```
Comment (nmcharlton) [inline src/components/Filter.js:268]: This can be simplified to `setTokenId(e.target.value)`.

File: src/components/FilterTop.js [hunk 4]
```diff
@@ -198,6 +201,36 @@ function Filter(props) {
                   </MenuItem>
                 ))}
               </TextField>
+              <TextField
+                select
+                label="Token Status"
+                value={
```
Comment (nmcharlton) [inline src/components/FilterTop.js:207]: This can be simplified to `value={tokenId}`.

File: src/components/FilterTop.js [hunk 5]
```diff
@@ -198,6 +201,36 @@ function Filter(props) {
...
+                  tokenId === filterOptionAll
+                    ? filterOptionAll
+                    : tokenId === tokenizationStates.TOKENIZED
+                    ? tokenizationStates.TOKENIZED
+                    : tokenizationStates.NOT_TOKENIZED
+                }
+                onChange={(e) => {
+                  setTokenId(
```
Comment (nmcharlton) [inline src/components/FilterTop.js:215]: This can be simplified to `setTokenId(e.target.value)`.

**Review by nmcharlton — COMMENTED** [review]
I've suggested a couple of code simplifications that take advantage of the recent change.

File: src/components/FilterTop.js (hunk 5 above)
Comment (VLuisa) [reply src/components/FilterTop.js]: Heh whoops, thanks for catching this!

**Review by nmcharlton — APPROVED** [review]
Cracking work, thanks @VLuisa!

**Pull request merged**